Assets/HoloToolkit/Build/Editor/BuildDeployPortal.cs

```diff
@@ -170,7 +170,7 @@ public static AppDetails QueryAppDetails(string baseAppName, ConnectInfo connect
                 AppList appList = JsonUtility.FromJson<AppList>(appListJSON);
                 for (int i = 0; i < appList.InstalledPackages.Length; ++i)
                 {
-                    string appName = appList.InstalledPackages[i].PackageFullName;
+                    string appName = appList.InstalledPackages[i].Name;
```

Comment:
I looked into this a bit and I'm not sure the proper way to pull the full package name out of the APPX. I assume it's embedded in there somewhere. 

The proposed change makes the naming consistent (always using the product name - it's mixed right now). If there are two applications with the same product name, it could still launch the wrong one (possible, as you point out). Currently, if someone fills out the product name, but not the package name (or it just doesn't match the product name), launch could fail - not great. 

If you know how to pull the full package name out of a built APPX, the full change should be pretty easy.